server/src/main/java/org/opensearch/common/settings/ClusterSettings.java

```diff
@@ -345,6 +345,8 @@ public void apply(Settings value, Settings current, Settings previous) {
             SearchService.DEFAULT_ALLOW_PARTIAL_SEARCH_RESULTS,
             ElectMasterService.DISCOVERY_ZEN_MINIMUM_MASTER_NODES_SETTING,
             TransportSearchAction.SHARD_COUNT_LIMIT_SETTING,
+            TransportSearchAction.SEARCH_REQUEST_CANCELLATION_ENABLE_SETTING,
+            TransportSearchAction.SEARCH_REQUEST_CANCEL_AFTER_TIMEINTERVAL_SETTING,
```

Comment:
There has been concern over the surface area of the settings getting too big. I think this design in general could even be done without the default cluster setting at this stage.

Reply:
Replied [here](https://github.com/opensearch-project/OpenSearch/pull/986#discussion_r681274801)